pubsub/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/pubsub/PubSubRef.scala

```diff
@@ -95,7 +95,7 @@ final class PubSubRef[T] private[lagom] (val topic: TopicId[T], mediator: ActorR
    * This method is especially useful when writing tests that require that a subscriber
    * is known before sending messages to a topic.
    */
-  def hasAnySubscribers(): Future[Boolean] = {
+  def hasAnySubscribers: Future[Boolean] = {
```

Comment:
The Scala style guides use case for omission of parentheses on zero arity methods is for pure functions - functions with no side effects. This function causes messages to be sent over the network, hence it rightly has parentheses.